index.html

```diff
@@ -0,0 +1,396 @@
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </main>
+    <div class="background-image">
+      <div class="background-tools">
+        <div
+          class="
+            form-style
+            d-flex
+            justify-content-center
+            flex-column flex-wrap
+            align-content-center
+          "
+        >
+          <div class="title">
+            <h2 class="text-white p-2">Contact me</h2>
+          </div>
+          <div class="content">
+            <p class="text-white">
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I'd
+              love to help you with that
+            </p>
+          </div>
+          <div>
+            <form
+              action="https://formspree.io/f/xknkkjzp"
+              method="POST"
+              class="p-2"
+            >
+              <div class="form-group">
+                <label class="text-white col-12">
+                  Your name:
+                  <input
+                    type="text"
+                    name="name"
+                    maxlength="50"
+                    class="form-control"
+                    placeholder="Uzair Ali"
+                  />
+                </label>
+              </div>
+              <div class="form-group">
+                <label class="text-white col-12">
+                  Your email:
+                  <input
+                    type="email"
+                    name="_replyto"
+                    class="form-control"
+                    placeholder="uzair@gmail.com"
+                  />
+                </label>
+              </div>
+              <div class="form-group">
+                <label class="text-white col-12">
+                  Your message:
+                  <textarea
+                    name="message"
+                    maxlength="500"
+                    class="form-control"
+                    rows="5"
+                    placeholder="Write your message here ....."
+                  ></textarea>
+                </label>
+              </div>
+              <!-- your other form fields go here -->
+              <button
+                type="submit"
+                class="btn btn-light text-primary col-4 button-margin"
```

Comment:
- [ ] Your submit button is looking great! Unfortunately, it needs to look as close as possible to the one in the template, so please consider styling the hover and the active pseudoclasses of the button to look like it. 😄 

Your button: 
![image](https://user-images.githubusercontent.com/73256642/131736236-94944791-e2a6-414e-ad41-c37c6cc62d83.png)
Template button:
![image](https://user-images.githubusercontent.com/73256642/131736362-77df8f15-9ee9-411c-b2d7-2db9d861d867.png)